mlflow/utils/_spark_utils.py

```diff
@@ -17,6 +17,23 @@ def _get_active_spark_session():
         return SparkSession._instantiatedSession
 
 
+def _create_spark_session():
```

Comment:
This should probably be renamed to be explicit in its purpose. `_create_local_spark_session_for_pipelines()` or something like that. And a solid docstring to explain its purpose since it implements a very restrictive functionality of an otherwise very common task (getting or creating a spark session).

Reply:
+1

Reply:
Thanks @BenWilson2 for the feedback. Updated.